### PR TITLE
initial commit of hello world file

### PR DESCRIPTION
**Pull request by mdleveille**
requesting to merge my hello world file with STScI's master branch

### CODE REVIEW

File: trainees/hello_044.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Hello world!
```
Comment (rs63) [inline trainees/hello_044.txt]: Consider using a capital 'W'.

Comment (rs63) [reply trainees/hello_044.txt]: consider using a Capital 'W'

Comment (rs63) [inline trainees/hello_044.txt]: Consider 'W'

Comment (mdleveille) [reply trainees/hello_044.txt]: I'd rather not

Comment (rs63) [reply trainees/hello_044.txt]: okay

**Pull request merged**